### DISCUSSION TITLE
Switch off check for updates

### DISCUSSION THREAD

**Discussion by ColaMix** [Open-Shell/Open-Shell-Menu#1905] [Q&A]
Hello together

We distribute openShell via software management. Is there a string or something that I give to the distribution that permanently disables the update check during installation.

**Comment by ge0rdi** [comment]
You can simply omit the `Update` component (feature) during installation.
For example, run installer with `ADDLOCAL=StartMenu` to install just Start menu component.

Eventually, you can disable update setting by running this command prior to installation:
`reg add HKLM\SOFTWARE\OpenShell\OpenShell /v Update /t REG_DWORD /d 0 /f`
